sdk/communication/communication-administration/src/communicationIdentity/communicationIdentityClient.ts

```diff
@@ -60,7 +62,17 @@ export class CommunicationIdentityClient {
     credential: KeyCredential,
     options?: CommunicationIdentityOptions
   );
-
+  /**
+   * Initializes a new instance of the CommunicationIdentity class using a TokenCredential.
+   * @param url The endpoint of the service (ex: https://contoso.eastus.communications.azure.net)
+   * @param credential An object that is used to authenticate requests to the service. Use the TokenCredential.
+   * @param options Optional. Options to configure the HTTP pipeline.
+   */
+  public constructor(
```

Comment:
We should add to the Changelog that we added a constructor that supports TokenCredential. 
We should also add using the constructor to samples/README later in another PR.